common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON

```diff
@@ -0,0 +1,1256 @@
+									  "<account_name>"
+								  ]
+							  }
+						  },
+						  "response": []
+					  },
+					  {
+						  "name": "delete account",
+						  "request": {
+							  "method": "DELETE",
+							  "header": [
+								  {
+									  "key": "Content-Type",
+									  "value": "application/json"
+								  },
+								  {
+									  "key": "X-chkp-sid",
+									  "value": "{{session}}"
+								  }
+							  ],
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/accounts/<account_name>",
+								  "host": [
+									  "{{server}}"
+								  ],
+								  "path": [
+									  "v1.8",
+									  "cme-api",
+									  "v1.2",
+									  "accounts",
+									  "<account_name>"
+								  ]
+							  }
+						  },
+						  "response": []
+					  }
+					]
+		},
+		{
+			"name" : "Gw Configurations",
+			"description" : "Operations for configuring Gw Configurations in CME <br/>(Previously known as \"templates\").",
+			"item": [
+					  {
+						  "name": "GET all gw configurations",
+						  "request": {
+							  "method": "GET",
+							  "header": [
+								  {
+									  "key": "X-chkp-sid",
+									  "value": "{{session}}",
+									  "type": "text"
+								  },
+								  {
+									  "key": "Content-Type",
+									  "value": "application/json",
+									  "type": "text"
+								  }
+							  ],
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations",
+								  "host": [
+									  "{{server}}"
+								  ],
+								  "path": [
+									  "v1.8",
+									  "cme-api",
+									  "v1.2",
+									  "gwConfigurations"
+								  ]
+							  }
+						  },
+						  "response": []
+					  },
+					  {
+						  "name": "GET a gw configuration",
+						  "request": {
+							  "method": "GET",
+							  "header": [
+								  {
+									  "key": "X-chkp-sid",
+									  "value": "{{session}}",
+									  "type": "text"
+								  },
+								  {
+									  "key": "Content-Type",
+									  "value": "application/json",
+									  "type": "text"
+								  }
+							  ],
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/<gwConfigurations_name>",
+								  "host": [
+									  "{{server}}"
+								  ],
+								  "path": [
+									  "v1.8",
+									  "cme-api",
+									  "v1.2",
+									  "gwConfigurations",
+									  "<gwConfigurations_name>"
+								  ]
+							  }
+						  },
+						  "response": []
+					  },
+					  {
+						  "name": "delete A gw configuration",
+						  "request": {
+							  "method": "DELETE",
+							  "header": [
+								  {
+									  "key": "X-chkp-sid",
+									  "type": "text",
+									  "value": "{{session}}"
+								  },
+								  {
+									  "key": "Content-Type",
+									  "type": "text",
+									  "value": "application/json"
+								  }
+							  ],
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/<gwConfigurations_name>",

```

Comment:
Please change to "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/<gwConfiguration_name>"